src/components/dialogs/parameters/dynamicsimulation/curve/common/checkbox-select.tsx

```diff
@@ -67,7 +78,7 @@ const CheckboxSelect = ({
     );
 
     const getOptionLabel = useCallback(
```

Comment:
why this definition, there is already a prop which is called `getOptionLabel` and defaulted to `defaultGetOptionLabel`

Reply:
YES, very strange this code, may be I forgot clean code after dev.. I will update in my proposed PR: https://github.com/gridsuite/gridstudy-app/pull/2418

Reply:
cleaned in this commit: https://github.com/gridsuite/gridstudy-app/pull/2418/commits/12d6b2319526f4f63d2eef154ebb9952e9c384bb

Reply:
I only changed all files to TS without trying to refactor code
I know it's tempting to fix all those small weird things, but it will make review even more difficult